Rakefile

```diff
@@ -252,6 +252,5 @@ end
 desc 'Run rubocop & integration tests for Image Streamer & specified path'
 task 'test:i3s:path', [:path] do |_t, spec|
   spec_pattern = spec['path']
-  Rake::Task[:rubocop].invoke
```

Comment:
Why did you remove rubocop?

Also, if it was really intended to be removed, you forgot to change the description of the task.